examples/end-to-end-examples/finetuning/utils_convert_to_mds.py

```diff
@@ -0,0 +1,269 @@
+# Copyright 2022 MosaicML Examples authors
```

Comment:
I think this script actually belongs in foundry with the other data prep scripts: https://github.com/mosaicml/llm-foundry/tree/main/scripts/data_prep